app/src/main/java/com/example/concordia_campus_guide/Models/Building.java

```diff
@@ -39,10 +43,9 @@
     private float bearing;
 
     //Not sure what to do here yet
-    //
-    //
-    @Ignore
-    private List<List<List<Double>>> coordinates;
+    @Embedded
```

Comment:
Did you try this? Because I read that u can't use @Embedded when you have the @Entity annotation on the class.

Reply:
Unless I misunderstood something.

Reply:
it works when I run it (and check in debug), so unless you have a link saying its not good practice, I think it's good